src/nuklear_text_editor.c

```diff
@@ -395,6 +395,7 @@ nk_textedit_text(struct nk_text_edit *state, const char *text, int total_len)
             {
                 nk_textedit_makeundo_insert(state, state->cursor, 1);
                 ++state->cursor;
+                state->cursor = NK_MIN(state->cursor, state->string.len);
```

Comment:
Maybe
```c
state->cursor = NK_MIN(state->cursor +1, state->string.len);
```
instead of
```c
++state->cursor;
state->cursor = NK_MIN(state->cursor, state->string.len);
```
would convey the message better? IDK

I will merge this PR after we settle on something :wink:.

Reply:
I like that shorthand. Made the change in the PR over at https://github.com/Immediate-Mode-UI/Nuklear/pull/486

Reply:
good point